litmus-portal/frontend/src/views/Community/CommunityTimeSeriesPlot/index.tsx

```diff
@@ -301,7 +301,7 @@ const CommunityAnalyticsPlot: React.FC = () => {
             y: 1,
             traceorder: 'normal',
             font: {
-              family: 'ubuntu',
+              family: 'Inter',
```

Comment:
Let's remove font-family entirely

Reply:
Plotly required explicit declaration of the font. cc: @ishanguta-ds

Reply:
yes @arkajyotiMukherjee

Reply:
Also this would not work if we don't add this font in litmus rather than just litmus-ui @whitetiger1399 @arkajyotiMukherjee